conf/messages.cy

```diff
@@ -642,6 +642,37 @@ registerInHomeCountry.p1=Dylech gofrestru ar gyfer GUC TAW gyda’r awdurdod tre
 registerInHomeCountry.p2=Os ydych eisiau cofrestru ar gyfer GUC TAW yn y DU, mae angen i chi
 registerInHomeCountry.callUs = ein ffonio.
 
+#######################################################
+##  DoYouWantToBePaidNetOrGross
+#######################################################
+doYouWantToBePaidNetOrGross.title = A ydych eisiau cael eich talu’n net neu’n gros?
+doYouWantToBePaidNetOrGross.heading = A ydych eisiau cael eich talu’n net neu’n gros?
+doYouWantToBePaidNetOrGross.subtext1 = Os cewch eich talu’n net, caiff 20% o’ch cyflog ei ddidynnu at ddibenion treth ac Yswiriant Gwladol.
+doYouWantToBePaidNetOrGross.subtext2 = Os ydych am gael eich talu’n gros, rhaid i chi {0} cyn mynd ymlaen
```

Comment:
Was it confirmed that this was the right place to put the anchor tag?

Reply:
Yes, as per ticket